### DISCUSSION TITLE
What's the behavior for one source with multiple sinks

### DISCUSSION THREAD

**Discussion by marcolan018** [vectordotdev/vector#10269] [Q&A] [answered]
Hi I have a question about the scenario below:
There is one source, and two sinks A and B consume the data from this source. But the sink A cannot work correctly. Then, what's the behavior for Vector?
1. will the data forwarded to sink B successfully?
2. when the external client post data to the source, will it always get error response because sink A failure?
Thanks.

**Comment by spencergilbert** [comment, marked as answer]
Each sink will receive a copy of an event and operate independently, so (1) will be true sink B will operate successfully. In terms of backpressure, if the failures of sink A cause Vector to apply backpressure it'll affect the throughput for both sinks utilizing the same source.
